go.mod

```diff
@@ -5,30 +5,40 @@ go 1.21.1
 toolchain go1.21.5
 
 require (
-	github.com/conduitio/conduit-commons v0.0.0-20231205181721-bef91d55116c
-	github.com/goccy/go-json v0.10.2
+	github.com/bufbuild/buf v1.28.1
+	github.com/conduitio/conduit-commons v0.0.0-20231222154604-f6dfae573b45
```

Comment:
Needs to be updated after https://github.com/ConduitIO/conduit-commons/pull/7 is merged.

Reply:
Updated.